src/Algebra/Morphism/Definitions.agda

```diff
@@ -23,11 +23,11 @@ open import Function.Core
 Morphism : Set _
 Morphism = A → B
```

Comment:
Should these `Morphism`s definition be deprecated if we are moving
away from them?

Reply:
That is certainly my opinion.  Making A and B invisible in a signature does not make it clearer.  And `Morphism` is longer than what it stands for!